File: Js/ex_01.js
/* Soma Simples
Escreva um programa que solicite dois números inteiros ao usuário e exiba a soma desses dois números. */

// Modulo instalado para receber dados do usuario
const readline = require('readline-sync');

function somaSimples(num1, num2) {
    return num1 + num2;
}

const numero1 = parseInt(readline.question("Digite o primeiro numero: "));
const numero2 = parseInt(readline.question("Digite o segundo numero: "));

console.log("A soma é: " + somaSimples(numero1, numero2));

File: Py/ex01.py
#Soma SimplesEscreva um programa que solicite dois números inteiros ao usuário e exiba a soma desses dois números.
#Modulo instalado para receber dados do usuario#


def calculo():
    user1 = int(input("Digite um numero: "))
    user2 = int(input("Digite outro numero: "))
    x = user1 + user2
    result = print(f" A soma entre esses numeros é  de {x}")

    return result


calculo()

